core/templates/pages/admin-page/navbar/admin-navbar.directive.html

```diff
@@ -18,11 +18,11 @@
   admin-navbar .oppia-navbar-admin {
     top: 0;
   }
-  admin-navbar .oppia-navbar-container {
-    background-color: #00376d !important;
+  admin-navbar #oppia-navbar-container {
```

Comment:
Here and elsewhere, do not use ID (or !Important). As mentioned, use a specific CSS selector if necessary, so that the appropriate rule gets triggered based on the CSS specificity rules. You may need to read up on how CSS selection works.

Reply:
Removed id.